sdk/identity/identity-cache-persistence/package.json

```
@@ -60,7 +60,7 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
-    "@azure/identity": "2.1.0-beta.3",
+    "@azure/identity": "2.1.0",
```

this should be a range right?
```suggestion
    "@azure/identity": "^2.1.0",
```